common/util/create_version_header.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+# Copyright 2020 The Verible Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+GIT_HASH=$(grep GIT_HASH bazel-out/volatile-status.txt | cut -f2 -d' ')
+
```

Comment:
Does this header generator need to produce `#include` guards as well?  Might not hurt.  I do realize it is only referenced as a private header in `srcs` of the BUILD file.

Reply:
Meh, I don't think it is worthwhile the visual clutter it would mean.